Docker/xrd-dev-stack/hurl-src/setup.hurl

```diff
@@ -47,7 +47,7 @@ Content-Type: application/json
   "password": "Secret1234"
 }
 
-HTTP 200
+HTTP *
```

Comment:
this is a workaround to allow setting up autologin for dev without breaking setup. In some cases autologin was triggering for me and this returned 409, since token was already active.  Alternative would have been to setup autologin with a separate bash script, which would just make things even more complex.